EXOSIMS/Prototypes/TargetList.py

```
@@ -342,6 +349,29 @@ def __init__(
         self.earths_only = bool(earths_only)
         self.scaleWAdMag = bool(scaleWAdMag)
         self.skipSaturationCalcs = bool(skipSaturationCalcs)
+        try:
+            massLuminosityRelationship = specs["modules"]["massLuminosityRelationship"]
+        except KeyError:
+            massLuminosityRelationship = "Henry1993"
+        self.massLuminosityRelationship = str(massLuminosityRelationship)
+        allowable_massLuminosityRelationships = [
+            "Henry1993",
+            "Fernandes2021",
+            "Henry1993+1999",
+            "Fang2010",
+        ]
+        self.allowable_massLuminosityRelationships = (
+            allowable_massLuminosityRelationships
+        )
+        if self.massLuminosityRelationship not in allowable_massLuminosityRelationships:

```

this should throw an error - remove this if and let the assert do its thing.